src/main/java/com/systemmeltdown/robotlib/subsystems/drive/TalonTrajectoryDriveSubsystem.java

```diff
@@ -135,20 +101,20 @@ public void resetOdometry(Pose2d pose) {
      */
     public DifferentialDriveWheelSpeeds getWheelSpeeds() {
         return new DifferentialDriveWheelSpeeds(m_leftEncoder.getRate(), m_rightEncoder.getRate());
-      }
-    
+    }
+
     /**
      * Controls the left and right sides of the drive directly with voltages.
      *
      * @param leftVolts  the commanded left output
      * @param rightVolts the commanded right output
      */
     public void setTankDriveVolts(double leftVolts, double rightVolts) {
-        m_leftTalonGroup.setMasterTalonVolts(leftVolts);
-        //rightVolts is negative because the right motors are inverted.
-        m_rightTalonGroup.setMasterTalonVolts(-rightVolts);
+        super.m_leftTalonMaster.setVoltage(leftVolts);
```

Comment:
SpeedControllerGroup has setVoltage, so you can call that instead of the masterTalon.